test/configCases/rule-set/test-user-fn/webpack.config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+	module: {
+		rules: [
+			{
+				test: /\.css$/,
+				// use: [{loader: 'style-loader'}, {loader: 'css-loader', options: {url: true}}],
+				use: info => {
+					console.log(info);
```

Comment:
Did you mean to leave this `console.log` here?

Reply:
just to see print